unresolved import `na::debug` when testing with "arbitraty" feature.

```rust
⋊> ~/t/p/r/nalgebra on master ◦ cargo test -j8 --features arbitrary                                                                 17:28:31
   Compiling nalgebra v0.16.0 (file:///home/brain/tmp/playground/rust-nalgebra/nalgebra)
error[E0432]: unresolved import `na::debug`
 --> tests/linalg/cholesky.rs:6:9
  |
6 | use na::debug::RandomSDP;
  |         ^^^^^ Could not find `debug` in `na`
```

--- a/ci/test.sh
+++ b/ci/test.sh
@@ -5,6 +5,7 @@ set -ev
 if [ -z "$NO_STD" ]; then
     if [ -z "$LAPACK" ]; then
         cargo test --verbose;
+        cargo test --verbose "arbitrary";
         cargo test --verbose "debug arbitrary mint serde-serialize abomonation-serialize";
     else
         cd nalgebra-lapack; cargo test --verbose;

--- a/tests/linalg/cholesky.rs
+++ b/tests/linalg/cholesky.rs
@@ -1,4 +1,4 @@
-#![cfg(feature = "arbitrary")]
+#![cfg(all(feature = "arbitrary", feature = "debug"))]
 
 use std::cmp;
 use na::{DMatrix, Matrix4x3, DVector, Vector4};

